pyproject.toml

```diff
@@ -67,23 +67,23 @@ python = ">=3.6,<3.9"
 coloredlogs = ">=10,<15"
 sanic = ">=19.12.2,<21.0.0"
 sanic-cors = "^0.10.0"
-requests = "^2.23"
+requests = ">=2.23.0,<2.26.0"
 
 [tool.poetry.dev-dependencies]
 pytest-cov = "^2.10.1"
-coveralls = "^2.1.2"
-pytest = "^5.4.1"
+coveralls = "^2.2.0"
+pytest = "^6.1.2"
```

Comment:
that's also different from Rasa Open Source which is fine, but I think for ease of development it would be great if these are similar